src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int NUMBER_OF_FIGURES = 5;
+    private static final int MAX_UNITS = 35;
```

Comment:
seems you've missed my [comment](https://github.com/mate-academy/jv-oop-advanced/pull/1212#discussion_r1288420786) :)